SPECS/golang/golang-1.15.spec

```diff
@@ -22,7 +22,6 @@ URL:            https://golang.org
 Source0:        https://dl.google.com/go/go%{version}.src.tar.gz
 Source1:        https://dl.google.com/go/go1.4-bootstrap-20171003.tar.gz
 Patch0:         go14_bootstrap_aarch64.patch
-Patch1:         CVE-2020-24553.patch
```

Comment:
We should remove the patch file, since it's no longer needed.

Reply:
removed